source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -221,7 +228,7 @@ The following example performs these actions:
 - Finds all documents with "Pizza" in the ``cuisine`` field
 - Sets the ``BatchSize`` to ``3``
 - Stores the results in a cursor
-- Prints the number of documents currently held in the cursor
+- Prints the documents in the cursor
```

Comment:
S: small technical wording fix, as the cursor is not a data structure but references documents 
```suggestion
- Prints the documents referenced by the cursor
```